wiki/Configure-lichess-bot.md

```diff
@@ -237,14 +237,19 @@ will precede the `go` command to start thinking with `sd 5`. The other `go_comma
-
-If there are entries for both real-time (`challenge_initial_time` and/or `challenge_increment`) and correspondence games (`challenge_days`), the challenge will be a random choice between the two.
 
-If there are entries for both absolute ratings (`opponent_min_rating` and `opponent_max_rating`) and rating difference (`opponent_rating_difference`), the rating difference takes precedence.
-
-The `challenge_filter` option can be useful if your matchmaking settings result in a lot of declined challenges. The bots that accept challenges will be challenged more often than those that have declined. The filter will remain until lichess-bot quits or the connection with lichess.org is reset.
+    The `challenge_filter` option can be useful if your matchmaking settings result in a lot of declined challenges. The bots that accept challenges will be challenged more often than those that have declined. The filter will remain until lichess-bot quits or the connection with lichess.org is reset.
+  - `block_list`: An indented list of usernames of bots that will not be challenged. If this option is not present, then the list is considered empty.
+  - `overrides`: Create variations on the matchmaking settings above for more specific circumstances. If there are any subsections under `overrides`, the settings below that will override the settings in the matchmaking section. Any settings that do not appear will be taken from the settings above. <br/>
+  The overrides section must have the following:
+    - Name: A unique name must be given for each override. In the example configuration below, `easy_chess960` and `no_pressure_correspondence` are arbitrary strings to name the subsections and they are unique.
+    - List of options: A list of options to override. Only the options mentioned will change when making the challenge. The rest will follow the default matchmaking options. In the example settings below, the blank settings for `challenge_initial_time` and `challenge_increment` under `no_pressure_correspondence` have the effect of deleting these settings, meaning that only correspondence games are possible.
+    
+    For each matchmaking challenge, the default settings and each override have equal probability of being chosen to create the challenge. For example, in the example configuration below, the default settings, `easy_chess960`, and `no_pressure_correspondence` all have a 1/3 chance of being used to create the next challenge. <br/>
```

Comment:
Replace the `<br/>` with an empty line below this one.

Reply:
Override section using newline:
![image](https://github.com/lichess-bot-devs/lichess-bot/assets/73843275/e643941f-baf4-44e8-b6da-5a728dbae4fa)
Override section using `<br/>`:
![image](https://github.com/lichess-bot-devs/lichess-bot/assets/73843275/1ca311f7-46b5-471b-a608-a6603d39f7fe)
This is why I used `<br/>`.

Reply:
I see. These should still be paragraph breaks with extra space, so how about adding a second `<br/>` on the same line in both places?